go.mod

```diff
@@ -1,4 +1,4 @@
-module ciam-rebac
+module relations-api
```

Comment:
Should this be the actual go import path? (e.g. github.com/project-kessel/relations-api ?) Or a domain?

This seems to be common in any case, so we might want to go with the github.com module name or like, project-kessel.org/relations-api or something like that

Reply:
github.com/project-kessel/relations-api  or project-kessel.org/relations-api (repo vs domain)

Decision: use: github.com/project-kessel/relations-api